dashboard/config/packages/old_sound_rabbit_mq.yaml

```diff
@@ -8,6 +8,6 @@ old_sound_rabbit_mq:
             enable_logger: true
             exchange_options:
                 name: 'dashboard'
-                type: direct
+                type: fanout
             queue_options:
-                name: 'dashboard'
+                name: 'articles'
```

Comment:
Should be the exchange's name changed?

Reply:
As we are not producing any messages (from what I can see) from the dashboard at present, I am not sure of the exchange's purpose here 🤔 .  Maybe was just for testing.

You only need the queue name (`articles` in this case) to receive the messages. The `exchange_options` entry can be removed if we are not producing messages from the dashboard.

Reply:
Should be the exchange that the queue is bound to (this is under the `consumers` config rather than producers).

Reply:
Can see this is actually working... taking a proper look.

Reply:
I had worked out what was happening, must not have submitted my follow-up. 🤔

This results in an exchange called 'dashboard' and a bound queue called 'articles', the article store _was_ creating an exchange called 'articles' and a bound queue called 'articles'. This works as the queues matched, the 'dashboad' exchange was just superfluous.

Now the article store doesn't create a queue, this probably no longer works. `exchange_options.name` needs to be  'articles' and `queue_options.name` needs to be 'dashboard'. That binds the queue for this app to the exchange provided by the article store.

Reply:
Ok, I have made the change and retested and it works fine.